src/runtime/audience-action-local-flow.ts

```diff
@@ -249,17 +193,21 @@ export class AudienceActionLocalFlow implements AudienceActionFlow {
       googletag.display(this.rewardedSlot_);
 
       // gpt.js has no way of knowing that an ad unit is invalid besides checking that rewardedSlotReady is called. Error out after 3 seconds of waiting.
-      setTimeout(() => {
-        if (!this.rewardedReadyCalled_) {
-          googletag.destroySlots([this.rewardedSlot_]);
-          resolve(false);
-        }
-      }, this.gptTimeout_);
+      setTimeout(this.rewardedAdTimeout_.bind(this), this.gptTimeout_);
     } else {
       resolve(false);
     }
   }
 
+  private async rewardedAdTimeout_() {
```

Comment:
It seems many of these functions are marked async unnecessarily (ie, no usage of await). Can you double-check?

Reply:
Done. Removed a lot of async.